sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/directconnectivity/ConsistencyWriter.java

```diff
@@ -6,9 +6,9 @@
 import com.azure.cosmos.BridgeInternal;
```

Comment:
Please consider sorting the imports here as one of them may now be out of order.

Reply:
Done